### PR TITLE
fix: Bug fixes calling UI

### PR DESCRIPTION
**Pull request by MarcinR**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability

--

Fix: ACC-421, ACC-422, ACC-423, ACC-424, ACC-361

--

ACC-421 Participants Cell layout is incorrect
ACC-422 Page indicator is not attached to  right side
ACC-423 User name and call status visible twice when someone you call someone
ACC-424 Participants list covered by notch
ACC-361 [iOS] Fix "pre calling UI" in the horizontal mode

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsView/CallingActionsView.swift [hunk 1]
```diff
@@ -99,8 +99,9 @@ class CallingActionsView: UIView {
     private func setupViews() {
         backgroundColor = .clear
         topStackView.distribution = .equalSpacing
-        topStackView.spacing = 6
-        verticalStackView.alignment = .center
+
+//        topStackView.spacing = 16
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsView/CallingActionsView.swift:103]: Is this needed?

Comment (agisilaos) [inline Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsView/CallingActionsView.swift:103]: Do we need that spacing after all?

File: Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallParticipantViews/CallParticipantDetailView.swift [hunk 2]
```diff
@@ -134,4 +133,11 @@ final class CallParticipantDetailsView: RoundedBlurView {
                                        withInsets: UIEdgeInsets.init(top: 4, left: 4, bottom: 4, right: 4))
         )
     }
+
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        guard previousTraitCollection?.userInterfaceStyle != traitCollection.userInterfaceStyle else { return }
+        microphoneImageView.image = StyleKitIcon.microphoneOff.makeImage(size: .tiny,
+                                                                         color: SemanticColors.Icon.foregroundMicrophone)
+    }
```
Comment (agisilaos) [inline Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallParticipantViews/CallParticipantDetailView.swift:142]: I'm not sure why do we need that here?

**Review by agisilaos — COMMENTED** [review]
Left some comments

File: Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallParticipantViews/CallParticipantDetailView.swift (hunk 2 above)
Comment (MarcinR) [reply Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallParticipantViews/CallParticipantDetailView.swift:142]: setting tint color on uiimageview didn't work so I have to create new picture when mode is changed

File: Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsView/CallingActionsView.swift (hunk 1 above)
Comment (MarcinR) [reply Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsView/CallingActionsView.swift]: removed

Comment (MarcinR) [reply Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingActionsView/CallingActionsView.swift]: removed

**Review by agisilaos — APPROVED** [review]

**Review by johnxnguyen — APPROVED** [review]

**Pull request merged**